tpm/verify.go

```diff
@@ -105,6 +105,16 @@ func (pk *pubKey) unmarshalRSAPublicKey() (*rsa.PublicKey, error) {
 	return rsapk, nil
 }
 
+// CreateQuoteInfo computes a quoteInfo structure for a given pair of data and PCR
+// values, packs the structure and then returns it.
+func CreateQuoteInfo(data []byte, pcrNums []int, pcrs []byte) ([]byte, error) {
+	qi, err := newQuoteInfo(data, pcrNums, pcrs)
+	if err != nil {
+		return nil, err
+	}
+	return tpmutil.Pack(qi)
+}
+
 // newQuoteInfo computes a quoteInfo structure for a given pair of data and PCR
 // values.
 func newQuoteInfo(data []byte, pcrNums []int, pcrs []byte) (*quoteInfo, error) {
```

Comment:
Make newQuoteInfo call tpmutil.Pack and expose that instead (maybe called `EncodeQuoteInfo`)
No need for a separate func.